CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - **Fix:** Prevent exception during calling `onError` asynchronously in `getUniqueId`.
 - **Update:** Supported "Update codebase" functionality for preview mode.
 - **Update:** Included technical improvement in custom permission and manifest information caching.
+- **Update:** Update `MessageToContact` to support an optional message to the user for the contact picker capability.
```

Comment:
@munir-rakuten @khairul-alam-licon This item was added to the wrong version - it should be under 3.X.X. We need to remember to move this before the release.

Reply:
Yeah, I'll add a separate PR for this.

Reply:
Yeah, I have addressed this issue in https://github.com/rakutentech/android-miniapp/pull/346
Thank you for noticing.